### DISCUSSION TITLE
The upload size limit on the upload proof page is wrong.

### DISCUSSION THREAD

**Discussion by nickchen120235** [community/community#108592] [GitHub Education]
### Hi everyone,

so I finally solved [my previous problem](https://github.com/orgs/community/discussions/106867). I checked the requests sent between me and GitHub by using Chrome DevTools and find out that the size limit of the upload is much smaller than what is stated on the page (100KB ~ 1MB). I uploaded a 7xxKB image and still got a 413 (Content Too Large) and redirected to the same page. I scaled down the image even more to 4xxKB and finally got approved.

GitHub should
1. check the size limit between the frontend and the backend
2. give user hint about what goes wrong during processing, not just redirecting to the same page, which confuses the users

**Comment by mishmanners** [comment]
The same is resolved here: https://github.com/orgs/community/discussions/88401#discussioncomment-8142514 and here: https://github.com/orgs/community/discussions/88446#discussioncomment-8148738

Please see these. The TL;DR:

- Refresh the page or clear your browser's cache and cookies
- Check for browser extensions that could be interfering with the JavaScript code and disable them
- Try a different browser
- Ensure your file is less than 150kB
